charts/pogues/templates/api/deployment.yaml

```diff
@@ -62,6 +62,8 @@ spec:
             value: {{ .Values.api.environment.authentification | quote}}
           - name: FR_INSEE_POGUES_ROLE_ADMIN
             value: {{ .Values.api.environment.role.admin | quote}}
+          - name: FR_INSEE_POGUES_STAMP_RESTRICTED
+            value: {{ .Values.api.environment.stamp.restricted | quote}}  
```

Comment:
Value that was added here should be indicated in `pogues/values.yaml` and in `bowie/values.yaml` (and in other charts that use the Pogues' chart)

If the value is missing we get a `nil pointer evaluating interface {}.restricted` error 💥